src/utils/getDataSetsKeys.js

```diff
@@ -0,0 +1,23 @@
+/*
+ * @flow
+ */
+
+import { List, Map, get } from 'immutable';
+import type { UUID } from 'lattice';
+
+export default function getDataSetsKeys(
+  dataSets :Map<UUID, Map>,
+  dataSetsColumns :Map<UUID, Map<UUID, Map>>,
+) :List<List<UUID>> {
+
+  return List().withMutations((keys :List<List<UUID>>) => {
+    dataSets.forEach((dataSet, dataSetId) => {
```

Comment:
Just being nitpicky here ... you only care about the key, so `.keySeq().forEach()` should be a little better because the `dataSet` object doesn't get brought into scope.